README.md

```diff
@@ -51,7 +51,7 @@ Each version of Cluster API for Azure will attempt to support at least two Kuber
 
 ## Documentation
 
-Documentation is in the `/docs` directory, and the [index is here](docs/README.md).
+Documentation is in the `/docs` directory, and the [index is here](https://github.com/kubernetes-sigs/cluster-api-provider-azure/blob/master/docs/README.md).
```

Comment:
this is the GitHub readme file and we injecting this in the book to build the main page.
So in order to render the links correctly in the book we need to add the public URL in the readme.md in the repository.

Reply:
there is one more (sorry!) 
`[flavors_doc]: ./templates/flavors/README.md` in the last line of this file

Reply:
oh missed that :( sorry.

done